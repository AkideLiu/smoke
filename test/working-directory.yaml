tests:
  - name: working-directory
    args:
      - fixtures/working-directory
    stdout:
      file: io/working-directory.out
