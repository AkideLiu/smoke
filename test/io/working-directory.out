global/set the working directory for the whole specification
  succeeded
global/override the working directory
  succeeded
local/set the working directory
  succeeded
local/set the working directory to another directory
  succeeded
local/resolve `..`
  succeeded

5 tests, 0 failures
